### PR TITLE
[Doc] Add a section on StarRocks alert rules and their handling methods

### PR DESCRIPTION
**Pull request by DanJing-coder**
## What I'm doing:

Add a section on StarRocks alert rules and their handling methods.

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [x] 3.2
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/en/administration/management/monitoring/alert.md [hunk 1]
```diff
@@ -0,0 +1,691 @@
+# StarRocks cluster alarm rules and handling methods
+
+This article introduces the alarm items that need attention and their handling methods from multiple dimensions such as business continuity, cluster availability, and machines.
+
+The following $job_name needs to be replaced with the corresponding job name in the prometheus configuration, and $fe_leader needs to be replaced with the corresponding ip of the leader fe.
```
Comment (DanRoscigno) [inline docs/en/administration/management/monitoring/alert.md:5]:
```suggestion
The following `$job_name` needs to be replaced with the corresponding job name in the prometheus configuration, and `$fe_leader` needs to be replaced with the corresponding IP of the leader FE.
```

File: docs/en/administration/management/monitoring/alert.md [hunk 2]
```diff
@@ -0,0 +1,691 @@
...
+
+Expression:
+
+```sql
+count(up{group="fe", job="$job_name"}) >= 3
+```
+
+Alarm description:
+
+Please adjust the number of surviving Fe nodes according to the actual number of Fe nodes.
```
Comment (DanRoscigno) [inline docs/en/administration/management/monitoring/alert.md:19]:
```suggestion
Please adjust the number of surviving FE nodes according to the actual number of FE nodes.
```

File: docs/en/administration/management/monitoring/alert.md [hunk 3]
```diff
@@ -0,0 +1,691 @@
...
+node_info{type="be_node_num", job="$job_name",state="dead"} > 1
+```
+
+Alarm description:
+
+The number of pending nodes are greater than 1.
+
+Processing method:
+
+There is a be service down, try to pull up.
```
Comment (DanRoscigno) [inline docs/en/administration/management/monitoring/alert.md:39]:
```suggestion
There is a BE service down, try to pull up.
```

File: docs/en/administration/management/monitoring/alert.md [hunk 4]
```diff
@@ -0,0 +1,691 @@
...
+(1-node_memory_MemAvailable_bytes{instance=~".*"}/node_memory_MemTotal_bytes{instance=~".*"})*100 > 90
+```
+
+Alarm description:
+
+Memory usage rate exceeds 90%.
+
+Processing method:
+
+Troubleshooting methods can refer to [memory problem troubleshooting](https://docs.starrocks.io/docs/administration/management/resource_management/Memory_management/) or get [heap profile](https://github.com/StarRocks/starrocks/pull/35322) troubleshooting.
```
Comment (DanRoscigno) [inline docs/en/administration/management/monitoring/alert.md:85]:
```suggestion
Troubleshooting methods can refer to [memory problem troubleshooting](./management/resource_management/Memory_management.md) or get [heap profile](https://github.com/StarRocks/starrocks/pull/35322) troubleshooting.
```

Comment (DanRoscigno) [reply docs/en/administration/management/monitoring/alert.md:85]: We need to copy the troubleshooting from that PR into the docs. @DanJing-coder can we add it in this doc, or should it be somewhere else?

File: docs/en/administration/management/monitoring/alert.md [hunk 5]
```diff
@@ -0,0 +1,691 @@
...
+
+Alarm description:
+
+Disk load over 90%
+
+Processing method:
+
+If there is a node_disk_io_time_seconds_total alarm, then first confirm whether there is a change in the business, if there is, whether the change can be rolled back, maintain the resource balance conditions before, if not, consider whether it is normal business growth leads to the need to expand resources.
+
+You can use the iotop tool to analyze and monitor disk I/O usage. Iotop has a UI similar to top, including pid, user, I/O, process and other related information.
```
Comment (DanRoscigno) [inline docs/en/administration/management/monitoring/alert.md:108]:
```suggestion
You can use the `iotop` tool to analyze and monitor disk I/O usage. Iotop has a UI similar to top, including pid, user, I/O, process and other related information.
```

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by yingtingdong — APPROVED** [review]

**Pull request merged**